AmericanMedicalAssociation/ama-style-guide-2
# ADF-276 Mobile Filters

**Jira Ticket**
- [ADF-276: Mobile Filters](https://palantir.atlassian.net/browse/ADF-276)

## Description

Create and style the mobile view for filter results according to the [design](https://app.zeplin.io/project/59dd1f9c36a8b21d569bb9e8/screen/6362d362f1e29ab52ce37a32)


## To Test

- Checkout the ‹ADF-276-mobile-filters branch and cd into style guide, run composer install

- Go to [DR Finder Filter Results Page](http://localhost:3000/?p=pages-dr-finder-with-search-results) 

- Inspect the page and switch to Mobile view, Do you see the `All Filters` button?

- Click on `All Filters` button, did you see the filter list menu show up?

- Verify the menu is working properly.

## Related Images

<img width="609" alt="Screen Shot 2022-11-09 at 4 15 40 PM" src="https://user-images.githubusercontent.com/54925022/200963372-f4ed193c-d2fc-4570-906b-99076649703a.png">
<img width="447" alt="Screen Shot 2022-11-09 at 4 15 09 PM" src="https://user-images.githubusercontent.com/54925022/200963396-01949836-b2db-484d-93b3-311699d20ae7.png">
